packages/core/admin/admin/src/content-manager/components/DynamicZone/components/Component/index.js

```diff
@@ -56,6 +58,21 @@ const Component = ({
 }) => {
   const { formatMessage } = useIntl();
   const { getComponentLayout } = useContentTypeLayout();
+  const { modifiedData } = useCMEditViewDataManager();
+  const componentLayoutData = useMemo(() => {
+    const layout = getComponentLayout(componentUid);
+
+    console.log('DZone/Component componentLayoutData', {
+      componentUid,
+      layout,
+      name,
+      index,
+      modifiedData,
+    });
+
+    return layout;
+  }, [componentUid, getComponentLayout, index, name, modifiedData]);
+  const mainValue = useMainValue(componentLayoutData, `${name}.${index}`);
```

Comment:
```suggestion
  const mainValue = useMainValue(componentLayoutData, [name, index]);
```

What do you think about not concatinating/ splitting the name and passing in the path instead?

Reply:
my mistake :), did not think it through...

Reply:
my mistake :), did not think it through...